package.json

```diff
@@ -47,15 +47,17 @@
     "@types/node": "20.3.1",
     "@types/react": "18.2.8",
     "@types/react-dom": "18.2.5",
-	"@typescript-eslint/eslint-plugin": "^7.6.0",
-	"@typescript-eslint/parser": "^7.6.0",
+    "@typescript-eslint/eslint-plugin": "^7.6.0",
+    "@typescript-eslint/parser": "^7.6.0",
     "encoding": "^0.1.13",
+    "eslint": "^9.0.0",
```

Comment:
Why is ESLINT being added to this PR?

Reply:
Perhaps it is because I've been doing `git pull origin develop` every day I continued working on this? Other than that, I've only used `pnpm install` when I originally created the branch.